ansible/roles/system/tasks/main.yml

```diff
@@ -206,15 +209,28 @@
     update_cache: true
   when: not cdefs_exist
 
-- name: Install Anthias dependencies
+- name: Get architecture
+  ansible.builtin.command: dpkg --print-architecture
```

Comment:
Use a more native way for this. You can get it from ansible directly. 

The long term plan is to be less Debian specific so the less hard coded we can be the better.

Reply:
@nicomiguelino this is what I mean

https://stackoverflow.com/questions/44713880/how-do-i-make-decision-based-on-arch-in-ansible-playbooks

Meaning, use "ansible_facts.ansible_architecture" instead of shelling out.

Reply:
Got it, thanks! Please see newer commits.